app/lib/actions/candlepin/environment/set_content.rb

```diff
@@ -52,7 +52,12 @@ def finalize # rubocop:disable Metrics/AbcSize
         def existing_ids
           ::Katello::Resources::Candlepin::Environment.
               find(input[:cp_environment_id])[:environmentContent].map do |content|
-            content[:content][:id]
+            # Supports Candlepin 4.2.13 and older versions
```

Comment:
Just a thought: should it prefer `contentId` since it's the modern way, and then fall back?

Reply:
Yeah that would improve readability a bit!